src/posix/platform/openthread-posix-config.h

```diff
@@ -229,4 +229,15 @@
 #error "OPENTHREAD_CONFIG_POSIX_TREL_USE_NETLINK_SOCKET was removed (no longer applicable with TREL over DNS-SD)."
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_POSIX_TREL_UDP_PORT
+ *
+ * This setting configures the TREL UDP port number.
+ * Define as 0 to use an ephemeral port number.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_POSIX_TREL_UDP_PORT
+#define OPENTHREAD_CONFIG_POSIX_TREL_UDP_PORT 0
```

Comment:
Should we unify to the same style like other configurations?
```suggestion
#ifndef OPENTHREAD_POSIX_CONFIG_TREL_UDP_PORT
#define OPENTHREAD_POSIX_CONFIG_TREL_UDP_PORT 0
```